zend_scoutapm.h

```diff
@@ -8,9 +8,7 @@
 #ifndef ZEND_SCOUTAPM_H
 #define ZEND_SCOUTAPM_H
 
-#ifdef HAVE_CONFIG_H
```

Comment:
What's the HAVE_CONFIG_H define?

Reply:
It is defined by autoconf, but since exts are always built with autoconf, this can be omitted (if the `#ifdef` is kept, it confuses CLion who thinks that `HAVE_CONFIG_H` is *not* defined)